Simplified/en.lproj/Localizable.strings

```diff
@@ -74,6 +75,7 @@
 "SyncFailed" = "Sync Failed";
 "TimedOut" = "The server did not respond. Please check your connection or try again later.";
 "TryAgain" = "Try Again";
+"Unknown error parsing problem document" = "Unknown error parsing problem document";
```

Comment:
It seems like the other string keys aren't space-separated. Maybe we should follow suit?

Reply:
we are sort of inconsistent there, for some strings we do use the space-separated English version as the key. I sort of prefer that approach because for incomplete localizations like our case, if a key is missing in a given localization you would at least see see the English version. Otherwise you see something like "UnknownErrorParsingProblemDocument" but in some cases it can be something really cryptic like "SettingsAccountViewControllerLogoutMessageDefault". I've seen this happening often for Italian for example.